### ISSUE TITLE
NVDA should announce first whether a thread is opened or closed.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#416]
**Reported by Hermann on 2009-09-16 13:15**
When you sort your mails/news/feeds by thread, it is important to know, 
whether a subject is a thread or a single news, and whether the thread is 
opened or closed.
In Thunderbird, but perhaps also in other mail clients, NVDA does not 
announce this when landing on a message in the messages list.
You can check out the thread status by activating "object position" in the 
objects preferences. However: You hear this position announcement 
everywhere, no matter if you want it or not; and moreover, it is announced 
at the end. So you have to listen to a long line in the messages list in 
order to hear whether you are on a thread and whether it is opened or 
closed.
So NVDA should announce "thread" if it is one, "open" and "closed" or 
"reduced" and "expanded".
Maybe this all belongs to the evergreen display hook, but perhaps the 
developers can find a solution before this famous feature is implemented.

**Comment by derekriemer** [comment]
No. This now works for me.
